cli/cmd/proxy/read/command_test.go

```diff
@@ -3,6 +3,7 @@ package read
 import (
 	"bytes"
```

Comment:
Should we have tests for some of the helper functions in command.go? such as fetchAdminPorts, fetchConfigs?

Reply:
Yes, I'll make a note of that. I feel they are tested enough to go in, at least through other PRs and whatnot. Are these tests a blocker for merging?